README.md

```diff
@@ -1,65 +1,121 @@
+5. Click “Show Advanced Settings” and we recommend setting both cameras as “Emulated”. Note that other camera configurations will work with the exception of setting both cameras as “webcam()”.
+  <img width="700px" src="images/emulator/emulator_avd_settings.png"/>
+6. Configure the rest of your device accordingly and click “Finish”.
+
+## Reducing APK Size
+
+Our library is built using native libriares. As a result, if you use the default gradle build you will generate an APK with all four architectures(armeabi-v7a, arm64-v8a, x86, x86_64 in your APK.
+
+[APK splits](https://developer.android.com/studio/build/configure-apk-splits.html) allow developers to build multiple APKs for different screen sizes and ABIs. Enabling APK splits ensures that the minimum amount of files required to support a particular device are packaged into an APK.
+
+The following snippet shows an example `build.gradle` with APK splits enabled.
+
+    apply plugin: 'com.android.application'
+    
+    android {
+        compileSdkVersion 24
+        buildToolsVersion "24.0.2"
+    
+        defaultConfig {
+            applicationId "com.twilio.video.quickstart"
+            minSdkVersion 16
+            targetSdkVersion 24
+            versionCode 1
+            versionName "1.0"
+        }
+    
+        // Specify that we want to split up the APK based on ABI
+        splits {
+            abi {
+                // Enable ABI split
+                enable true
+    
+                // Clear list of ABIs
+                reset()
+    
+                // Specify each architecture currently supported by the Video SDK
+                include "armeabi-v7a", "arm64-v8a", "x86", "x86_64"
+    
+                // Specify that we do not want an additional universal SDK
+                universalApk false
+            }
+        }
+    }
+    
+    dependencies {
+        compile "com.twilio:video-android:1.0.0-beta4"
+    }
+
+The adoption of APK splits requires developers to submit multiple APKs to the Play Store. Refer to [Google’s documentation](https://developer.android.com/google/play/publishing/multiple-apks.html) for how to support this in your application.
+
+## Setup an Access Token Server
+
+Using Twilio's Video client within your applications requires an access token. 
+
+You can run your own server that provides access tokens, based on your Twilio credentials. This server can either run locally on your development machine, or it can be installed on a server. If you run the server on your local machine, you should use the ngrok utility to give the server an externally accessible web address. That way, you can run the quickstart app on an Android device.
+
+### Configuring the Access Token Server
 
 If you want to be a little closer to a real environment, you can download one of the video quickstart applications - for instance, [Video Quickstart: PHP](https://github.com/TwilioDevEd/video-quickstart-php) and either run it locally, or install it on a server.
 
```

Comment:
I'll keep it for now but we can consider dropping this. I'd actually like to single source this across all our SDKs at some point and make it language agnostic, so not just have the php link.